cmd/eib/main.go

```diff
@@ -251,5 +249,19 @@ func bootstrapRpmDependencyServices(ctx *image.Context) bool {
 		ctx.RPMRepoCreator = rpm.NewRepoCreator(ctx.BuildDir)
 	}
 
+	if ctx.ImageDefinition.Kubernetes.Version != "" {
+		c, err := cache.New(rootDir)
+		if err != nil {
+			audit.AuditInfof("Failed to initialise file caching. %s", checkLogMessage)
```

Comment:
I was going back and forth with this one. I kept it like this because users would expect that caching is in place between builds so if this fails for whatever reason, then it will not only be plausible for the entire build to fail as you've said but we'd also have to do workarounds for the `cache` not being nil. It was too much hassle (at this point at least).